refactor directory structure

shiodat opened the pull request:
Related to issue #287, I changed the directory structure to follow the table of contents.

shiodat commented:
I fixed the conflicts and refactor English docs.

kmaehashi commented:
* I think it is better to name the directory index file as ``index.rst``, e.g., ``/ja/overview/index.rst`` instead of ``/ja/overview/summary.rst`` so that users can access it like ``http://jubat.us/ja/overview/``.

kmaehashi commented:
@shiodat
* Please update methods page in English as discussed.
* Please resolve the conflict.

shiodat commented:
@kmaehashi 
- I renamed the index files , e.g. `/ja/jubaql/jubaql.rst` <- `/ja/jubaql/index.rst`.
- I removed the Japanese descriptions at English methods page `en/method.rst`.
- I resolved the conflicts.

kmaehashi commented on source/en/fv_convert.rst:
```diff
@@ -534,7 +534,7 @@ It depends on "type" how to specify weight and name features.
     ============= =====================
     ``"num"``     Use given number itself as weight.
-    ``"log"``     Use natural logarithm of given number as weight. If the number is not positive, weight is 0.
+    ``"log"``     Use logarithm of given number as weight. If the number is not positive, weight is 0.
```
Degraded when resolving conflict?

shiodat replied:
I resolved conflicts again and fix the degradation in `fv_converter`.

kmaehashi commented on source/ja/fv_convert.rst:
```diff
@@ -630,7 +630,7 @@ string_rulesなどと同様、複数の規則を羅列する。
    ============ =====================
    ``"num"``    与えられた数値をそのまま重みに利用する。
-   ``"log"``    与えられた数値の自然対数を重みに利用する。但し、数値が1以下の場合は0とする。
+   ``"log"``    与えられた数値の対数を重みに利用する。但し、数値が1以下の場合は0とする。
```
Degraded when resolving conflict?

shiodat replied:
I resolved conflicts again and fix the degradation in `fv_converter`.

shiodat commented:
I resolved conflicts again and fix the degradation in `fv_converter`.
I used the old version of `fv_converter` and degredation occured, sorry.

kmaehashi commented:
:+1: Thank you!
